common/src/main/java/com/vexsoftware/votifier/support/forwarding/cache/FileVoteCache.java

```diff
@@ -67,11 +75,11 @@ private void load() throws IOException {
         JsonObject players = object.getAsJsonObject("players");
         JsonObject servers = object.getAsJsonObject("servers");
 
-        for (String player : players.keySet()) {
+        for (String player : keySet(players)) {
```

Comment:
Not saying its wrong, but why was this changed?

Reply:
See #93 for more information, but essentially this didn't work on BungeeCord because BungeeCord has an older GSON version (2.8.0) that lacks `keySet()` on `JsonObject`.